torch/quantization/observer.py

```diff
@@ -605,8 +605,8 @@ def calculate_qparams(self):
     def get_tensor_value(self):
         return self.tensor_val
 
-
 # Restrict activations to be in the range (0,127)
 default_observer = MinMaxObserver.with_args(reduce_range=True)
 default_debug_observer = RecordingObserver
 default_weight_observer = MinMaxObserver.with_args(dtype=torch.qint8, qscheme=torch.per_tensor_symmetric)
+default_histogram_observer = HistogramObserver.with_args(reduce_range=True)
```

Comment:
should it be "per architecture" defaults?

Reply:
do you mean having default_histogram_observer_fbgemm and default_histogram_observer_qnnpack?

Reply:
That comes in a later diff, where we specify an architecture to obtain a qconfig.
https://github.com/pytorch/pytorch/pull/26627